api/v1beta1/cluster_types.go

```diff
@@ -1160,7 +1160,7 @@ func ipFamilyForCIDRStrings(cidrs []string) (ClusterIPFamily, error) {
 }
 
 // ClusterIPFamily defines the types of supported IP families.
-type ClusterIPFamily int
+type ClusterIPFamily int32
```

Comment:
We can't change the existing APIs. These are all breaking changes

In my opinion we have to exclude specifically the current occurences of that finding and then we can fix it in newer apiVersions

(@JoelSpeed please correct me if I'm wrong)

Reply:
Oh, sorry. As you can said, this change affects compatibility.
Thanks.

@sbueringer 
Question. Does package exp allow us to change the type ? (e.g. exp/ipam/api/(v1alpha1|v1beta1)/ipaddress_types.go)

Reply:
IMHO no we should not do it there either, it should be done there when going to a new api version instead.

Reply:
You can see that this not only changes the go type (which can be fixed I guess?) but it also changes the schema which is generally a pretty good sign that we shouldn't make this change yet

Let's ignore these until we bump the API version

Reply:
> IMHO no we should not do it there either, it should be done there when going to a new api version instead.

Was that referring to cmd/clusterctl/api/v1alpha3/metadata_type.go?
That is only the type we use to parse metadata.yaml files.

Reply:
Fixed. Thanks all :)